launcher/src/Instance.scala

```diff
@@ -103,16 +106,43 @@ object Instance{
             case _ => ???
           }
         case multiple =>
-          throw new Exception(
-            s"More than one running instance found: ${multiple.map(_.instanceId()).mkString(", ")}. You can delete extra instances in the aws-dev-admin console (double check your region!)."
-          )
+          val instanceOptions = multiple.zipWithIndex.map(t => {
+            val (instance, idx) = t
+            val dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss z").withZone(ZoneId.systemDefault())
+            s"\t[${idx + 1}] ${instance.instanceId()} (${instance.placement().availabilityZone()}) launched at ${dateFormatter.format(instance.launchTime())}"
+          }).mkString("\n")
+          log(s"Multiple Devbox instances found. Which one would you like to keep? (enter 0 to discard all and get a new one)\n" + instanceOptions)
+          val chosen = scala.io.StdIn.readInt() - 1
```

Comment:
This might throw if the user types something other than an integer, perhaps put this in a loop? It doesn't matter much for this particular instance, as I guess it's easy to restart.